src/VisualStudio/Core/Def/Implementation/Venus/ContainedLanguage.cs

```diff
@@ -69,53 +68,6 @@ internal partial class ContainedLanguage<TPackage, TLanguageService>
         // flickering.
         private ITagAggregator<ITag> _bufferTagAggregator;
 
-        [Obsolete("This is a compatibility shim for TypeScript and Live Share; please do not use it.")]
-        public ContainedLanguage(
-            IVsTextBufferCoordinator bufferCoordinator,
-            IComponentModel componentModel,
-            AbstractProject project,
-            IVsHierarchy hierarchy,
-            uint itemid,
-            TLanguageService languageService,
-            SourceCodeKind sourceCodeKind,
-            IFormattingRule vbHelperFormattingRule,
-            Workspace workspace)
-            : this(bufferCoordinator,
-                   componentModel,
-                   project.VisualStudioProject,
-                   hierarchy,
-                   itemid,
-                   project.ProjectTracker,
-                   project.Id,
-                   languageService,
-                   vbHelperFormattingRule: null)
-        {
-            Contract.ThrowIfFalse(vbHelperFormattingRule != null);
```

Comment:
📝 I botched this assertion in db2227e525f23669b37ab57316055390fb809ea2, which functionally broke this constructor in the 16.1 branch. If we can get @uniqueiniquity to fix the TypeScript cases and someone to fix the Live Share cases, we can avoid the need to fix this assertion and still later need to remove the constructors.